eq-author/src/App/settings/ThemesPage.js

```diff
@@ -241,7 +239,7 @@ const ThemesPage = ({ questionnaire }) => {
                         onBlur={(e) => handleBlur({ ...e.target })}
                         data-test="change-questionnaire-id"
                         errorValidationMsg={surveyIdError}
-                        errorAlignLeft
+                        errorAlignRight={false}
```

Comment:
```suggestion
```

The absence of errorAlignRight in the props will have the value within the StyledInput component be false.